mybuilder/mybuilder.bzl

```diff
@@ -23,6 +23,14 @@ def mybuilder_repositories(
         server_urls = maven_servers,
     )
 
+    jvm_maven_import_external(
+        name = "mybuilder_rules_picocli",
+        artifact = "info.picocli:picocli:3.9.5",
+        #jar_sha256 = "9bc4992d3b7d98885a2c42c301654f54b13a5747",
```

Comment:
I'll try and compute this after downloading the jar, if not ill defer to bazel to do it.